docs/pages/eas-update/eas-cli.mdx

```diff
@@ -109,6 +109,16 @@ Renaming branches do not disconnect any channel–branch links. If you had a cha
   cmdCopy="eas branch:rename --from [branch-name] --to [branch-name]"
 />
 
+### Connect a chanel with a branch 
+
+you can connect any channel with a branch by running 
+
+```
+eas channel:edit
+```
+
+This command will help you to map the branches with the channles that youh ave
```

Comment:
Hey @a0m0rajab it seems like typo mistake here as here is `channles` also the spacing between the `youh ave`.

Thanks

Reply:
@sharmayatendra thanks, just fixed it.